packages/components/config/playwright/setup/utils/Stickersheet.ts

```diff
@@ -133,18 +133,23 @@ class StickerSheet {
    * Mounts components onto the page using specified combinations of attributes.
    * @param combinations - An object where keys are attribute names and
    * values are objects containing possible attribute configurations.
+   * @param createNewRow - A boolean (optional) indicating whether to create a new row for each combination.
+   * Its default value is false.
    * @throws Will throw an error if tagname is not defined.
    */
 
-  public async mountComponents(combinations: Record<string, Record<string, any>>) {
+  public async mountComponents(combinations: Record<string, Record<string, any>>, createNewRow: boolean = false) {
```

Comment:
Do we need to set type when we assign the value at declaration ?
```suggestion
  public async mountComponents(combinations: Record<string, Record<string, any>>, createNewRow = false) {
```